Doppler.Currency.Test/CurrencyServiceTests.cs

```diff
@@ -26,6 +27,8 @@ public class CurrencyServiceTests
         private readonly Mock<IHttpClientFactory> _httpClientFactoryMock;
         private readonly Mock<HttpMessageHandler> _httpMessageHandlerMock;
         private readonly HttpClient _httpClient;
+        private readonly Mock<ICurrencyFactory> _currencyFactoryMock;
+        private readonly Mock<CurrencyHandler> _currencyMockHandler;
```

Comment:
I think that this variable is not used, can you remove it?

Reply:
updated